druid-shell/src/platform/windows/window.rs

```diff
@@ -161,6 +161,7 @@ struct WndState {
     render_target: Option<DeviceContext>,
     dcomp_state: Option<DCompState>,
     dpi: f32,
+    min_size: Option<kurbo::Size>,
```

Comment:
`Size` is already imported from `kurbo` so this can be changed to just `Option<Size>`.

Reply:
Good point, fixed